### PR TITLE
Translated 2014-09-03-introducing-the-jsx-specification.md to spanish

### PR DESCRIPTION
**Pull request by FLiotta**
Translated the 2019-09-03-introducing-the-jsx-specification.md #201

### CODE REVIEW

**Comment by FLiotta** [comment]
@alejandronanez it doesn't let me edit reviewers to assign you

**Comment by carburo** [comment]
@FLiotta Yes, don't worry. You don't need to assign a reviewer.

**Comment by FLiotta** [comment]
Nice! ty :)

**Comment by alejandronanez** [comment]
Thanks for the PR! 

I'll take a look at this on Saturday. 👏🏼

File: content/blog/2014-09-03-introducing-the-jsx-specification.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 ---
-title: "Introducing the JSX Specification"
+title: "Introduciendo el concepto de JSX"
```
Comment (carburo) [inline content/blog/2014-09-03-introducing-the-jsx-specification.md:2]:
```suggestion
title: "Introduciendo la especificación de JSX"
```

File: content/blog/2014-09-03-introducing-the-jsx-specification.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 author: [sebmarkbage]
 ---
 
-At Facebook we've been using JSX for a long time. We originally introduced it to the world last year alongside React, but we actually used it in another form before that to create native DOM nodes. We've also seen some similar efforts grow out of our work in order to be used with other libraries and in interesting ways. At this point React JSX is just one of many implementations.
+En Facebook hemos estado usando JSX por un largo tiempo. Originalmente lo introdujimos al mundo el ultimo año junto a React, pero en realidad lo usábamos de otra forma antes de usarlo para crear nodos nativos del DOM.
+También hemos visto esfuerzos similares surgir de nuestro trabajo para ser utilizado con otras librarías de maneras interesantes. En este punto React JSX es solo una de muchas implementaciones.
```
Comment (carburo) [inline content/blog/2014-09-03-introducing-the-jsx-specification.md:7]:
```suggestion
En Facebook hemos estado usando JSX por un largo tiempo. Originalmente lo introdujimos al mundo el año pasado junto con React, pero en realidad lo usábamos de otra forma anteriormente para crear nodos nativos del DOM. También hemos visto esfuerzos similares surgir de nuestro trabajo para ser utilizado con otras bibliotecas de maneras interesantes. En este punto React JSX es solo una de muchas implementaciones.
```

File: content/blog/2014-09-03-introducing-the-jsx-specification.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
-In order to make it easier to implement new versions and to make sure that the syntax remains compatible, we're now formalizing the syntax of JSX in a stand-alone spec without any semantic meaning. It's completely stand-alone from React itself.
+Para que sea más fácil de implementar nuevas versiones y asegurar que la sintaxis se mantenga compatible, estamos formalizando la sintaxis de JSX como un funcionamiento independiente sin ningún significado semántico. Es completamente independiente de React en si.
```
Comment (carburo) [inline content/blog/2014-09-03-introducing-the-jsx-specification.md:9]:
```suggestion
Para que sea más fácil de implementar nuevas versiones y asegurar que la sintaxis se mantenga compatible, estamos formalizando la sintaxis de JSX como una especificación independiente sin ningún significado semántico. Es completamente independiente del propio React.
```

File: content/blog/2014-09-03-introducing-the-jsx-specification.md [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
 
-Read the spec now at <https://facebook.github.io/jsx/>.
+Lee las especificaciones ahora en <https://facebook.github.io/jsx/>.
```
Comment (carburo) [inline content/blog/2014-09-03-introducing-the-jsx-specification.md:11]:
```suggestion
Lee la especificación ahora en <https://facebook.github.io/jsx/>.
```

File: content/blog/2014-09-03-introducing-the-jsx-specification.md [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
 
-This is not a proposal to be standardized in ECMAScript. It's just a reference document that transpiler writers and syntax highlighters can agree on. It's currently in a draft stage and will probably continue to be a living document.
+Esta no es una propuesta para estandarizarlo en ECMAScript. Es solo un documento de referencia en donde los escritores de transpiladores y los resaltadores de sintaxis puedan concordar. Esta actualmente en una etapa de boceto pero probablemente continúe hasta convertirse en un documento.
```
Comment (carburo) [inline content/blog/2014-09-03-introducing-the-jsx-specification.md:13]:
```suggestion
Esta no es una propuesta que deba estandarizarse en ECMAScript. Es solo un documento de referencia en donde los escritores de transpiladores y los resaltadores de sintaxis puedan concordar. Está actualmente en una etapa de boceto pero probablemente continúe hasta convertirse en un documento.
```

**Review by carburo — CHANGES_REQUESTED** [review]

**Review by carburo — APPROVED** [review]

**Pull request merged**